bin/nova-hypervisor-usage

```diff
@@ -192,18 +196,35 @@ def compute_capability(flavor, hypervisors):
     max_count_wo_disabled = 0
 
     for node in hypervisors:
-        vms_by_cpus = int((node.vcpus_max - node.vcpus_used) / flavor.vcpus)
         vms_by_mem = node.memory_mb_free / flavor.ram
-        n_vms = min(
-            vms_by_cpus,
-            vms_by_mem)
-        if n_vms < 0: n_vms = 0
-
-        max_vms_by_cpus = int((node.vcpus_max) / flavor.vcpus)
         max_vms_by_mem = node.memory_mb / flavor.ram
-        n_maxvms = min(
-            max_vms_by_cpus,
-            max_vms_by_mem)
+
+        if 'gpu' in flavor.name:
+            if 'gpuT4' in flavor.name:
+                n_maxvms = min(
+                    DEFAULT['GPU_T4_SLOTS'],
+                    max_vms_by_mem
+                    )
+            elif 'gpuP4' in flavor.name:
+                n_maxvms = min(
+                    DEFAULT['GPU_P4_SLOTS'],
+                    max_vms_by_mem
+                    )
+            #else throw exception
+            n_vms = min(
+                compute_pci_slots_node_capability(n_maxvms, node.running_vms),
+                vms_by_mem
+                )
```

Comment:
Following the comment on compute_pci_slots_node_capability() 
```suggestion
             n_avail_slots = n_maxvms - node.running_vms    
             n_vms = min(
                n_avail_slots,
                vms_by_mem
                )
```

Reply:
I thought it would, as there could be more complicated / automated ways to calculate that.
I was most probably wrong.